camelot/view/action_steps/select_object.py

```diff
@@ -110,10 +92,15 @@ class SelectObjects( OpenTableView ):
     def __init__(self, admin, search_text=None, value=None):
         if value is None:
             value = admin.get_query()
-        select_admin = SelectAdminDecorator(admin)
-        super(SelectObjects, self).__init__(select_admin, value)
+        super(SelectObjects, self).__init__(admin, value)
```

Comment:
nu w er hier wel geen ReadOnly decoration meer toegepast op de lijst.

op zich is dat voor onze use case waarschijnlijk best wel ok nu, we moeten er gewoon 'aware' voor zijn.

zou dit ook zo laten, omwille van : 

- ReadOnlyDecorator moet toch verdwijnen
- Als we de lijst weergave overzetten naar QML is hier zowiezo geen editing meer in